src/main/java/com/contrastsecurity/core/extended/ExtendedContrastSDK.java

```diff
@@ -37,19 +37,25 @@
     private static final int SERVER_ERROR = 500;
     private Gson gson;
     private String restApiURL;
+    private IntegrationName integrationName;
```

Comment:
why don't you just use the super classes Integration name and version?

Reply:
So right now, I wasn't sure if I'd need to use those later in the class like we do for the restApiUrl for makeRequest. Because we will likely add those in the base class "makeRequest" and not the overriden one, they will probably be removed. 

That being said, we aren't leveraging the builder class in the Extended sdk, which seems like a waste. So I will revise that to include that. In order to do that, I will need to add a getter to the sdk for the restApi variable (at the least), as we reference that directly.

Reply:
I made some changes to the sdk that will help with this https://github.com/Contrast-Security-OSS/contrast-sdk-java/pull/64